entrypoint.sh

```diff
@@ -21,13 +21,11 @@ if [[ -n "${SONAR_ROOT_CERT}" ]]; then
 fi
 
 if [[ -f "${INPUT_PROJECTBASEDIR%/}/pom.xml" ]]; then
-  echo "Maven project detected. You should run the goal 'org.sonarsource.scanner.maven:sonar' during build rather than using this GitHub Action."
-  exit 1
+  echo "WARNING! Maven project detected, you might have only partial analysis. You can run the goal 'org.sonarsource.scanner.maven:sonar' during build instead of this GitHub Action."
```

Comment:
```suggestion
  echo "WARNING! Maven project detected. Sonar recommends running the 'org.sonarsource.scanner.maven:sonar' goal during the build process instead of using this GitHub Action to get more accurate results."
```